jssource/src_files/include/javascript/sugar_3.js

```diff
@@ -799,22 +799,24 @@ function add_error_style(formname, input, txt, flash) {
           window.setTimeout('fade_error_style(style, ' + wp * 10 + ')', 1000 + (wp * 50));
         }
       }
-      if (typeof (window[formname + "_tabs"]) != "undefined") {
-        var tabView = window[formname + "_tabs"];
-        var parentDiv = YAHOO.util.Dom.getAncestorByTagName(inputHandle, "div");
-        if (tabView.get) {
-          var tabs = tabView.get("tabs");
-          for (var i in tabs) {
-            if (tabs[i].get("contentEl") == parentDiv
-              || YAHOO.util.Dom.isAncestor(tabs[i].get("contentEl"), inputHandle)) {
-              tabs[i].get("labelEl").style.color = "red";
-              if (inputsWithErrors.length == 1)
-                tabView.selectTab(i);
+      if (typeof window[formname + "_tabs"] != "undefined") {
+        if ($(".validation-message")) {
+          // Reset tab error styles
+          $("#EditView_tabs li a").removeAttr("style");
+          counter = 0;
+          $(".validation-message").each(function () {
+            // Get error validation tabs ids
+            CurrentValidationTabId = $(this).closest(".tab-pane-NOBOOTSTRAPTOGGLER").attr("id").split('-').slice(-1);
```

Comment:
Hi @SinergiaCRM, 2 small thing:
- don't we need to use `var` here like `var currentValidationTabId`?
- Also the variable names should start with lowecase.

Reply:
Done